ionic2/app/pages/about/about.page.scss

```diff
@@ -1,3 +1,7 @@
 .about {
 
 }
+
+ion-item a{
+  text-decoration: none;
```

Comment:
I'd prefer it to be in the scss file by default just like the template should be in a html file unless there are very good reasons to not do it. Also isn't finding out more exciting than just playing it safe 😉 . Also you will not have to play Sherlock Holmes to find the styling you are looking for.